pkg/config/config.go

```diff
@@ -256,3 +257,14 @@ func getKeyPtrs(aMap map[string]ParameterStoreVal) []*string {
 	}
 	return keys
 }
+
+// AccountSvc returns the account Service for you
+func (config *ConfigurationBuilder) AccountSvc() accountiface.Servicer {
+
+	var accountSvc accountiface.Servicer
+	if err := config.GetService(&accountSvc); err != nil {
+		panic(err)
```

Comment:
Left this as a Panic so the dot notation could be used.  Instead doing (accountiface.Servicer, error) we can just return the appropriate service allowing for someone to do `config.AccountSvc().Update(...)`